modules/oxide/src/less/theme/components/toolbar-button/toolbar-number-input.less

```diff
@@ -0,0 +1,43 @@
+//
+// Toolbar number input
+//
+
+@toolbar-number-input-focus-color: #d9edf7;
```

Comment:
ok, I tried to copy the toolbar-button color style, I don't know if it would me more correct to add the class `tox-tbtn` and then create a more specific identifier for width and height